src/test/java/com/slcube/shop/business/cart/controller/CartControllerTest.java

```diff
@@ -0,0 +1,135 @@
+package com.slcube.shop.business.cart.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.slcube.shop.business.cart.dto.CartListResponseDto;
+import com.slcube.shop.business.cart.dto.CartSaveRequestDto;
+import com.slcube.shop.business.cart.dto.CartUpdateRequestDto;
+import com.slcube.shop.business.cart.service.CartService;
+import com.slcube.shop.common.security.WithMockMember;
+import com.slcube.shop.common.security.authenticationContext.MemberContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.*;
+import static org.mockito.BDDMockito.*;
+import static org.springframework.http.MediaType.*;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@WebMvcTest(CartController.class)
+@WithMockMember
+class CartControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private CartService cartService;
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private MemberContext memberContext;
+
+    @BeforeEach
+    void setUp() {
+        memberContext = (MemberContext) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+    }
+
+    @Test
+    @DisplayName("장바구니 저장")
+    void saveCartTest() throws Exception {
+        CartSaveRequestDto requestDto = new CartSaveRequestDto();
+        requestDto.setItemId(1L);
+        requestDto.setQuantity(10);
+
+        given(cartService.saveCart(requestDto, memberContext.getMember()))
+                .willReturn(1L);
+
```

Comment:
```suggestion

        given(cartService.saveCart(any(CartSaveRequestDto.class), any(memberContext.class)))
                .willReturn(1L);

```
테스트 코드 또한 관리 포인트 중에 하나입니다 그리고 테스트 코드 관리에서 특정 파라미터를 지정하면 나중에 파라미터 하나하나 다 관리해야해서 어려움이 있을 수 있습니다
그러므로 저는 any 를 넣어서 아무 값이나 들어와도 저렇게 반환 될 수 있도록 하는걸 추천해요 만약 a 값을 입력하면 A가 나오고 b값을 입력하면 B가 나올 경우에만 저는 특정 값을 넣어 주는 형태로 개발을 합니다